.github/workflows/issue-comment-created.yml

```
@@ -7,8 +7,13 @@ on:
 env:
   IN_MAINTAINER_LIST: ${{ contains(secrets.MAINTAINER_LIST, github.actor) }}
 
+permissions:
+  contents: read
+
 jobs:
   issue_comment_triage:
+    permissions:
+      contents: none
```

In order to remove labels, we need write permissions

```suggestion
    permissions:
      issues: write
```